closure/goog/i18n/numberformat.js

```diff
@@ -462,9 +462,9 @@ goog.i18n.NumberFormat.prototype.parseNumber_ = function(text, pos) {
       normalizedText += '.';
       sawDecimal = true;
     } else if (ch == grouping.charAt(0) &&
-               ('\u00a0' != grouping.charAt(0) ||
-                pos[0] + 1 < text.length &&
-                this.getDigit_(text.charAt(pos[0] + 1)) >= 0)) {
+        ('\u00a0' != grouping.charAt(0) ||
+        pos[0] + 1 < text.length &&
+        this.getDigit_(text.charAt(pos[0] + 1)) >= 0)) {
```

Comment:
How? Run gjslint? I always run `gjslint --strict --jslint_error=all <file>` every time I check in and push.

I did it again now and viola, we're good. Do you want me to revert it back to what we had?